.github/workflows/update-db-csv-files.yml

```diff
@@ -0,0 +1,70 @@
+name: Update DB files
```

Comment:
Sage Monorepo hosts multiple products. Consider adding information that associate this workflow to OpenChallenges. E.g. in workflow title and/or job name.